java/src/test/java/ome/jxrlib/TestInMemoryDecode.java

```diff
@@ -99,27 +131,31 @@ public void testByteArray(
             throws IOException, DecodeException {
         byte[] data = asByteArray(filename);
 
-        TestDecode decode = new TestDecode(data);
-        assertDecode(decode, width, height, bpp, md5);
+        assertDecode(data, width, height, bpp, md5);
     }
 
     @Parameters({"filename", "md5"})
     @Test
     public void testFirstFrame(String filename, String md5)
             throws IOException, DecodeException {
         byte[] data = asByteArray(filename);
-        byte[] destination = TestDecode.decodeFirstFrame(data, 0, data.length);
+        byte[] destination = new TestDecode().decodeFrame(0, data);
         Assert.assertEquals(md5, md5(destination));
     }
 
+    /*
```

Comment:
Let's add a TestNG annotation here and comment as disabled for now or adjust the test cases to re-use a ByteBuffer if it's a memory issue we're running into.